pwiz_tools/Skyline/Util/Install.cs

```diff
@@ -153,10 +155,11 @@ public static string ProgramNameAndVersion
         {
             get
             {
-                return string.Format(@"{0} ({1}-bit{2}{3}) {4}",
+                return string.Format(@"{0} ({1}-bit{2}{3}{4}) {5}",

```

Comment:
Seems like this has an extra space? " {5}" with possible replacements for {5} leading to " " or "  running on Wine" (two spaces)

Reply:
No extra space (" running on Wine" was assigned to {4} inside the parens, not {5}).